Uploadable/MimeTypeGuesserAdapter.php

```diff
@@ -2,14 +2,14 @@
 
 namespace Stof\DoctrineExtensionsBundle\Uploadable;
 
-use Symfony\Component\HttpFoundation\File\MimeType\MimeTypeGuesser;
 use Gedmo\Uploadable\MimeType\MimeTypeGuesserInterface;
+use Symfony\Component\Mime\MimeTypes;
```

Comment:
Unlike HttpFoundation, Mime isn't a required dependency of FrameworkBundle. You must add it explicitly as a dependency in `composer.json`, or throw in `guess()` if it isn't available.

Reply:
I've added symfony/mime to composer.json suggest section